models/content-type-snippets/index.ts

```diff
@@ -1 +1 @@
-export * from './sEOMetadata';
+export * from './SEOMetadata';
```

Comment:
this file does not exists :\ therefore the build fails

Reply:
I had some problems with case-sensitivity of Git. It should be fixed now.